README.org

```diff
@@ -4,14 +4,24 @@
   by the community for the community. In broad strokes, this repository exists for
   the following purposes in roughly the order they're listed.
 
+  - Allow Matrix community to easily document the current behavior of implementations
+    in a specification format, when such behaviour deviates from the upstream spec.
+    - Speed up the process of adoption and validation of new features/endpoints.
+    - Give a more bleeding-edge reference document that Matrix community can use to
+      prepare for stable releases of the official specification.
+    - Avoid duplication of work and effort in reverse-engineering deviation from
+      outdated specification.
   - Allow Matrix community to easily document pieces of the de-facto Matrix spec
     which are either undocumented in the official spec or are unclear.
     - To facilitate this documentation occurring as quickly as possible, this kind
       of documentation is not required to be in any specific format. Document the
-      spec however you like (.md, .doc, .txt, whatever).
+      spec however you like (.md, .doc, .txt, whatever). Official spec format is still
```

Comment:
I'm not sure what this means? We don't have the whole setup the official docs does

Reply:
While we would accept other formats than the "official" documents format, it would make it harder to merge back into upstream and there would be overhead to process such documents and make them "ready".

I would like to say that, while we accept any format, we still would like contributions to follow the official format if possible.

Reply:
How about the compromise that the `matrix-doc` repo should stay as close to official spec format as possible, but `eventually-doc` can have more "freedom" to, as the README says, "facilitate this [...] occuring as quickly as possible"?

Reply:
That's what I'm trying to say yes. How would you word it?

Reply:
That last sentence could be "However, within the `matrix-doc` repository, please follow the existing style as closely as possible, as it will make it easier to hopefully(?) merge those changes back."

Reply:
But I would like it to apply to any repo we would fork! not just `matrix-doc`

Reply:
Sure, as long as we get to keep `eventually-doc` relatively "loose"

Reply:
New proposal: "Within forked repositories, please try to match your style to upstream to ease future merging."

Reply:
Very nice!